cinema/serializers.py

```diff
@@ -114,7 +143,15 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
 
     class Meta:
         model = MovieSession
-        fields = ("id", "show_time", "movie", "cinema_hall", "taken_places")
+        fields = (
+            "id",
+            "show_time",
+            "movie",
+            "cinema_hall",
+            "taken_places",
+            "image"
```

Comment:
The `image` field should not be included in the `MovieSessionDetailSerializer` fields list as it is not part of the `MovieSession` model.